apis/bases/core.openstack.org_openstackcontrolplanes.yaml

```diff
@@ -7701,6 +7701,22 @@ spec:
                     - secret
                     type: object
                 type: object
+              lightspeed:
+                properties:
+                  enabled:
+                    default: false
+                    type: boolean
+                  llm_credentials:
+                    type: string
+                  llm_endpoint:
+                    type: string
+                  tlsCert:
```

Comment:
Suggest using "tlsCACert", which is used in other code to clarify that the cert for validation, not serving or mTLS.

Reply:
Ack, thanks @csibbitt!